terminal.h

```diff
@@ -79,7 +79,7 @@ class Terminal : public QObject
 
     QList<TermChar>& currentLine();
 
-    Q_INVOKABLE void keyPress(int key, int modifiers);
+    Q_INVOKABLE void keyPress(int key, int modifiers, QString text);
```

Comment:
Minor detail, but commonly const reference. With default value can avoid "" in virtual keyboard handling.